lib/src/main/java/grpcbridge/route/UrlPathAndQuery.java

```diff
@@ -64,7 +64,8 @@ public String path() {
                     ? new SimpleImmutableEntry<>(keyAndValue[0], singletonList(""))
                     : new SimpleImmutableEntry<>(
                             keyAndValue[0],
-                            singletonList(URLDecoder.decode(keyAndValue[1], UTF_8.name())));
+                            singletonList(URLDecoder.decode(
```

Comment:
What does this do? Is there a built in function that does this?

Reply:
It replaces `+` to `%2B` which is the url encoded version of `+`, so later the decoder will decode it back to `+`. If we don't do this, the decoder will convert the `+` to a space character, that is just the behavior of URLDecoder. I am not sure but I doubt it that there is an option to turn that feature off.

Reply:
What about https://google.github.io/guava/releases/19.0/api/docs/com/google/common/net/UrlEscapers.html?